nextflow_schema.json

```diff
@@ -79,6 +79,26 @@
                     "fa_icon": "fas fa-ban",
                     "hidden": true,
                     "help_text": "Do not load `igenomes.config` when running the pipeline. You may choose this option if you observe clashes between custom parameters and those supplied in `igenomes.config`."
+                },
+                "bwamem2": {
+                    "type": "string",
+                    "format": "file-path",
+                    "fa_icon": "fas fa-copy",
+                    "description": "Path to directory for pre-built bwamem2 index.",
+                    "help_text": "> **NB** If none provided, will be generated automatically from the FASTA reference."
+                },
+                "save_reference": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "If generated by the pipeline save the required indices/references in the results directory.",
+                    "help_text": "The saved references can be used for future pipeline runs, reducing processing times.",
+                    "fa_icon": "fas fa-save"
+                },
+                "local_genomes": {
+                    "type": "string",
+                    "fa_icon": "fas fa-map-marker-alt",
+                    "description": "Directory / URL base for genomes references.",
+                    "help_text": "All files are supposed to be in the same folder defining a flat structure"
```

Comment:
I added these manually but it can be built using [web interface](https://nf-co.re/pipeline_schema_builder). These are necessary for display information in CLI log stdout and validation of parameter.